homeassistant/components/sms/const.py

```diff
@@ -27,3 +36,61 @@
     {"value": "76800", "label": "76800"},
     {"value": "115200", "label": "115200"},
 ]
+
+SIGNAL_SENSORS: Final[dict[str, SensorEntityDescription]] = {
```

Comment:
Please move the sensor entity descriptions to the sensor platform. These are platform details.